### PR TITLE
Cleaning playlists bonanza - The return

### PR DESCRIPTION
**Pull request by BellezaEmporium**
About AD : Port 1935 is an Adobe proprietary link for Flash, they're going to the same places as the HTTPS links underneath them. Better keep HTTPS links I suppose.

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
No need to keep this open furthermore. Cleaning links will be done simulteanously within the Broken Link verification.

However, as long as I haven't moved the new links, keep my branch open.

**Comment by Dum4G** [comment]
Would it be hard to reopen this pull req and resolve conflicts to merge everything you've done here?

**Comment by LaneSh4d0w** [comment]
> Would it be hard to reopen this pull req and resolve conflicts to merge everything you've done here?

I don't think so. I'll check in a few minutes

**Comment by LaneSh4d0w** [comment]
@Dum4G All good mate, fixed conflicts !

File: channels/am.m3u [hunk 1]
```diff
@@ -7,8 +7,6 @@ http://ott-cdn.ucom.am/s10/index.m3u8
 #EXTINF:-1 tvg-id="5TV.am" tvg-name="5TV" tvg-country="AM" tvg-language="Armenian" tvg-logo="https://i.imgur.com/yigw9dr.png" group-title="",5TV
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (iPhone; CPU iPhone OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148
 http://ott-cdn.ucom.am/s6/04.m3u8
-#EXTINF:-1 tvg-id="Amga.am" tvg-name="Amga" tvg-country="AM" tvg-language="Armenian" tvg-logo="https://i.imgur.com/3vQVwUV.jpg" group-title="",Amga
-https://streamer1.connectto.com/AMGA_WEB_1202/tracks-v1a1/mono.m3u8
```
Comment (Dum4G) [inline channels/am.m3u:11]: That works

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/us.m3u [hunk 2]
```diff
@@ -548,6 +548,8 @@ https://bonappetit-samsung.amagi.tv/playlist.m3u8
 https://condenast-bonappetit-vizio.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="bonappetit.us" tvg-name="bon appétit" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/YhFFxlE.png" group-title="Cooking",bon appétit
 https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxbonappetit/CDN/playlist.m3u8
+#EXTINF:-1 tvg-id="BonAppetit.us" tvg-name="Bon Appetit" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/YhFFxlE.png" group-title="",Bon Appetit
+https://condenast-bonappetit-vizio.amagi.tv/playlist.m3u8
```
Comment (Dum4G) [inline channels/us.m3u:552]: duplicate of line 548

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/us.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply channels/us.m3u:552]: Yeah I wonder why the "Incoming change" on conflict re-added this line.

**Review by Dum4G — APPROVED** [review]

**Pull request merged**